optimum/exporters/onnx/convert.py

```diff
@@ -264,17 +265,27 @@ def validate_model_outputs(
     ref_outputs = reference_model(**reference_model_inputs)
     ref_outputs_dict = {}
 
+    if isinstance(ref_outputs, dict):
+        outputs = ref_outputs.items()
+    elif isinstance(ref_outputs, (list, tuple)):
+        outputs = zip(list(config.outputs.keys()), ref_outputs)
+    else:
+        name = list(config.outputs.keys())[0]
+        ref_outputs_dict[name] = ref_outputs
+        outputs = []
```

Comment:
same comment as in model_patcher.py, can you comment add a comment as to why the if/elif/else is needed?